### ISSUE TITLE
Plans on continue development

### ISSUE THREAD

**Issue by woto** [tabler/tabler#898]
Sorry for inconvenience but may I ask a question. Is there any plans to support development of this product in future? :)

**Comment by dheineman** [comment]
@codecalm Can you please provide some input regarding the continued development of tabler? It looks like all development in this repository grinded to a halt over 2 months ago with complete radio silence from the maintainers.

**Comment by rjd22** [comment]
I've tried contacting @codecalm to offer to help out since I have a lot of patches ready that fixes a lot of bugs in dark mode but it has been radio silence from his side. Maybe he's really busy with something to taking a break from opensource for a while.

I'm not sure if @FreexD knows more about the situation.

I'm willing to fork this repo and add offer the patches. But I rather wanted to wait for some answers because forking is essentially splitting the community.

**Comment by NotNelson** [comment]
> I've tried contacting @codecalm to offer to help out since I have a lot of patches ready that fixes a lot of bugs in dark mode but it has been radio silence from his side. Maybe he's really busy with something to taking a break from opensource for a while.
> 
> I'm not sure if @FreexD knows more about the situation.
> 
> I'm willing to fork this repo and add offer the patches. But I rather wanted to wait for some answers because forking is essentially splitting the community.

I think the fork is looking more and more appealing at this point. Pawel's contribution record for the last 6 months tells as story as well... No reason not to fork it now and perhaps merge back into this repo _ala_ Fomantic/Semantic

**Comment by rjd22** [comment]
@NotNelson Yep it really saddens me but since @codecalm isn't responding not even to email I'm planning to fork this since I want to also maintain it for my client.

Since forking this is not completely straightforward I will need some time to see how to setup the building and releasing.

**Comment by Jamesking56** [comment]
@codecalm made some commits in https://github.com/tabler/static a few weeks ago, so maybe just ignoring us all?

**Comment by dheineman** [comment]
> @codecalm made some commits in https://github.com/tabler/static a few weeks ago, so maybe just ignoring us all?

I think that might be an automated task given the commit messages and times but then again their profile does indicate numerous contributions in private repositories over the past months

**Comment by rjd22** [comment]
@Jamesking56 @dheineman I checked those a while ago and at least the public ones are automated processes.

**Comment by Jamesking56** [comment]
This is a real shame as I really like this theme and want to use it extensively in my work. I think a fork is going to be the only way forward, is there someone here with some familiarity with Tabler that can setup a fork?

**Comment by sagegwatkin** [comment]
Sorry to hear about your father @codecalm.

I'm happy to help out on this project.

We compile the sass from source for each project, and have a bunch of fixes I can create pull requests for. I'm also able to respond to issues/discussions & review pull requests when I have time.
